Lib/pdb.py

```diff
@@ -1258,14 +1258,13 @@ def _getval_except(self, arg, frame=None):
                 return eval(arg, self.curframe.f_globals, self.curframe_locals)
             else:
                 return eval(arg, frame.f_globals, frame.f_locals)
-        except:
-            exc_info = sys.exc_info()[:2]
-            err = traceback.format_exception_only(*exc_info)[-1].strip()
+        except Exception as exc:
```

Comment:
The original was equivalent to `except BaseException as exc:` -- why not keep that?